### PR TITLE
fix effusive anima accelerator handling for monk/shaman

### PR DESCRIPTION
**Pull request by emallson**

### CODE REVIEW

File: src/parser/shadowlands/modules/covenants/kyrian/EffusiveAnimaAccelerator.ts [hunk 1]
```diff
@@ -11,17 +11,17 @@ interface SpellInfo {
...
   [CLASSES.DRUID]: SPELLS.KINDRED_SPIRITS,
   [CLASSES.HUNTER]: SPELLS.RESONATING_ARROW,
   [CLASSES.MAGE]: SPELLS.RADIANT_SPARK,
-  [CLASSES.MONK]: SPELLS.WEAPONS_OF_ORDER,
+  [CLASSES.MONK]: SPELLS.WEAPONS_OF_ORDER_CAST,
   [CLASSES.PALADIN]: SPELLS.DIVINE_TOLL,
   [CLASSES.PRIEST]: SPELLS.BOON_OF_THE_ASCENDED,
   [CLASSES.ROGUE]: SPELLS.ECHOING_REPRIMAND,
-  [CLASSES.SHAMAN]: SPELLS.VESPER_TOTEM,
+  [CLASSES.SHAMAN]: undefined,
```
Comment (abelito75) [inline src/parser/shadowlands/modules/covenants/kyrian/EffusiveAnimaAccelerator.ts]: If you want to add the spell it should be 324386

Comment (emallson) [reply src/parser/shadowlands/modules/covenants/kyrian/EffusiveAnimaAccelerator.ts]: its 1am and i wasn't planning to be on rn so i'm just YOLOing that through once i confirm tests pass. can get that fixed once someone wants to do rsham things

Comment (emallson) [reply src/parser/shadowlands/modules/covenants/kyrian/EffusiveAnimaAccelerator.ts]: actually, fuck it, i'll do it right

Comment (abelito75) [reply src/parser/shadowlands/modules/covenants/kyrian/EffusiveAnimaAccelerator.ts]: Yeah. I was just putting it out there if you wanted to do it haha
Its late just go to sleep haha

Comment (abelito75) [reply src/parser/shadowlands/modules/covenants/kyrian/EffusiveAnimaAccelerator.ts]: image
it appears i've been sniped

**Pull request merged**